css/example/Rating.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../components/Rating.css">
+    <script src="https://kit.fontawesome.com/dcbd873a92.js" crossorigin="anonymous"></script>
```

Comment:
Rather than using a script for fontawsome,you can use cdn link in your style sheet

Reply:
done